includes/settings.php

```diff
@@ -125,7 +125,8 @@ function update_notice( $plugin_file, $plugin_data, $status ) {
  * @since 1.2
  */
 function maybe_notice() {
-	if ( 0 === strpos( get_current_screen()->parent_base, 'distributor' ) ) {
+	$parent_base = get_current_screen()->parent_base ?? '';
```

Comment:
Rather than continuing execution if the parent_base isn't defined, I think the function can return early as it's already known that the condition won't pass.

```suggestion
	$parent_base = get_current_screen()->parent_base;
	if ( ! $parent_base ) {
		return;
	}
```